src/components/JobWizard/JobWizardSteps/JobWizardHyperparameterStrategy/JobWizardHyperparameterStrategy.js

```diff
@@ -44,10 +44,11 @@ const JobWizardHyperparameterStrategy = ({ formState }) => {
       </div>
       <div className="grid-container">
         <div className="strategy-grid-item">
-          <FormSelect label="Strategy" name={strategiesPath} options={selectOptions.strategies} />
+          <FormSelect id="hyperparams-stategy" label="Strategy" name={strategiesPath} options={selectOptions.strategies} />
```

Comment:
Can be applied the same logic as in the FormInput (using `name` prop inside the component as an `id`)

Reply:
It is the same logic. I don't use the "name" prop for the ID since I don't want the ID with "." char.